client/beefy/src/lib.rs

```diff
@@ -106,13 +107,14 @@ where
 }
 
 /// BEEFY gadget initialization parameters.
-pub struct BeefyParams<B, BE, C, N>
+pub struct BeefyParams<B, BE, C, N, SO>
 where
 	B: Block,
 	BE: Backend<B>,
 	C: Client<B, BE>,
 	C::Api: BeefyApi<B>,
 	N: GossipNetwork<B> + Clone + Send + 'static,
+	SO: SyncOracle + Send + Sync + Clone + 'static,
```

Comment:
Would these two ever be provided by different objects? If not, I think it would make sense to simply squash them together, they're both mandatory anyway...